backend/app/services/archive_draft_enrollments.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class ArchiveDraftEnrollments
```

Comment:
This service should be a query object imo.
A query object should return an activerecord relation, which can be manipulate with more query methods.

You can make a single SQL query here btw. If you want you can use https://www.db-fiddle.com/ to build you query.

Moreover, it's a bad name: this service does not archive draft enrollments, it sorts enrollments draft that can be archived

=> `DraftEnrollmentsArchiveable`

Reply:
This service file is supposed to be delete. 
I've made another service so we could archive enrollments according different status

I will check your link to improve the query methods